Cargo.toml

```diff
@@ -41,3 +41,4 @@ tower-http = { version = "0.4.3", features = ["cors"] }
 include_dir = "0.7.3"
 steamlocate = { version = "1.2.1", features = ["steamid_ng"] }
 watchman_client ={ version = "0.8.0" }
+zune-inflate = "0.2.54"
```

Comment:
Did you mean to add this?

Reply:
cant imagine where that would have come from, surely rust analyser doesnt automatically just add things to fricken cargo.toml. that seems wrong anyhway because its showing steamlocate 1.2 not 2 beta

Reply:
but yes i cant imagine what zune-inflate does

Reply:
> cant imagine where that would have come from, surely rust analyser doesnt automatically just add things to fricken cargo.toml

Must have accidentally `cargo add`ed it in the wrong directory when doing your other work lol

> that seems wrong anyhway because its showing steamlocate 1.2 not 2 beta

Probably because I selected the code on the first commit rather than the final thing, so I guess it doesn't show the final commit's changes.